src/quantum/plugin/QuantumOptions.ts

```diff
@@ -222,7 +222,10 @@ export class QuantumOptions {
         return this.optsTarget === "electron";
     }
     public isTargetUniveral() {
-        return this.optsTarget === "universal";
+        return this.optsTarget === "universal" || this.optsTarget === "npm";

```

Comment:
Ok, lemme test it manually (it's quite an important addition), I will get back to you, later on, today, thanks!